crates/hyfetch/src/presets.rs

```diff
@@ -343,6 +383,10 @@ impl Preset {
                 "#8869A5", "#D8A7D8", "#FFFFFF", "#FDB18D", "#151638",
             ]),
 
+            Self::Biromantic2 => ColorProfile::from_hex_colors(vec![
```

Comment:
Please remove this. This was just mislabeled greysexual flag, so it has been renamed in this rewrite.

Reply:
Ohhh okay, I see.